src/vs/workbench/contrib/comments/browser/commentsViewActions.ts

```diff
@@ -17,17 +17,23 @@ import { COMMENTS_VIEW_ID } from 'vs/workbench/contrib/comments/browser/comments
 import { FocusedViewContext } from 'vs/workbench/common/contextkeys';
 import { viewFilterSubmenu } from 'vs/workbench/browser/parts/views/viewFilter';
 
+export type CommentsSortOrder = 'resource' | 'updatedAt';
+
 const CONTEXT_KEY_SHOW_RESOLVED = new RawContextKey<boolean>('commentsView.showResolvedFilter', true);
 const CONTEXT_KEY_SHOW_UNRESOLVED = new RawContextKey<boolean>('commentsView.showUnResolvedFilter', true);
+const CONTEXT_KEY_SORT_BY = new RawContextKey<CommentsSortOrder>('commentsView.sortBy', 'resource');
```

Comment:
Preparing for supporting both ascending & descending sorting here. Not sure if there's any conventions in the codebase regarding sorting/options like this?